src/main/groovy/au/org/ala/bie/Blacklist.groovy

```diff
@@ -0,0 +1,111 @@
+package au.org.ala.bie
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.ObjectMapper
+import java.util.regex.Pattern
+
+/**
+ * A list of source information that should be
+ * blacklisted.
+ */
+
+class Blacklist {
+    /** Blacklist metadata */
+    @JsonProperty
+    Metadata metadata
+    /** The source URLs that should be black-listed */
+    @JsonProperty
+    List<Rule> blacklist
+
+    /**
+     * Test an article against the blacklist
+     * <p>
+     * If this returns true then the article has been blacklisted
+     *
+     * @param name The taxon name
+     * @param source The article source
+     * @param title The article title
+     *
+     * @return {@code true} if the input argument matches the predicate,
+     * otherwise {@code false}
+     */
+    boolean test(String name, String source, String title) {
```

Comment:
Still think the method can be named isBlacklisted()

Reply:
Done